rules/align.smk

```diff
@@ -13,7 +13,8 @@ rule align:
     output:
         # see STAR manual for additional output files
         "star/{sample}-{unit}/Aligned.out.bam",
-        "star/{sample}-{unit}/ReadsPerGene.out.tab"
+        "star/{sample}-{unit}/ReadsPerGene.out.tab",
+	"star/{sample}-{unit}/Log.final.out"
```

Comment:
This additional log file, shouldn't it be added to the log directive? What does it contain?

Reply:
thats alright, it can be removed. These logs were input to multiqc to ensure the align steps has finished before multiqc is run. I will remove it. I have a question in multiqc below.

Reply:
If they are needed for multiqc as input though, they should be kept.